main-app/app/scripts/gameboard/gameboard-services/human-logic-service.js

```diff
@@ -0,0 +1,14 @@
+angular.module('Server.MyModule')
+    .service('BothHumans', ['CoreData','FreshGame',  function (coreData, freshGame) {
```

Comment:
This is part of the game model responsibility, it isn't going to change so no need to plug.